rewrites.js

```diff
@@ -159,7 +159,8 @@ exports.REWRITES = [
     destination: '/orders',
   },
   {
-    source: '/:parentCollectiveSlug?/:collectiveType(events|projects)?/:collectiveSlug/orders/:OrderId([0-9]+)',
+    source:
+      '/:parentCollectiveSlug?/:collectiveType(events|projects)?/:collectiveSlug/(orders|contribution)/:OrderId([0-9]+)',
```

Comment:
To keep it consistent (e.g. `/expenses/4242`)

```suggestion
      '/:parentCollectiveSlug?/:collectiveType(events|projects)?/:collectiveSlug/(orders|contributions)/:OrderId([0-9]+)',
```